pkg/didcomm/protocol/issuecredential/states.go

```diff
@@ -331,12 +331,19 @@ func (s *offerReceived) ExecuteInbound(md *MetaData) (state, stateAction, error)
 		return nil, nil, fmt.Errorf("decode: %w", err)
 	}
 
+	response := &RequestCredential{
+		Type:           RequestCredentialMsgType,
+		RequestsAttach: offer.OffersAttach,
+	}
+
+	if md.RequestCredential() != nil {
```

Comment:
quick workaround - @soluchok should probably take a look at this

Reply:
@llorllale it seems to be fine.
This logic was added by @pfeairheller. We missed this part. Would be nice to cover it with tests (TODO is fine).
https://github.com/hyperledger/aries-framework-go/commit/d4d3721fdf9ca694188e3f15b00204e4fffdbb1e#diff-86393b3003ab06a03e9ec28f2f603b63688888c366300c4dcf28b557d39fe9b1